README.md

```diff
@@ -14,56 +14,122 @@
+
 **app.module.ts**
 
 ```typescript
 import {
-  EventStoreModule,
-  EventStore,
-  EventBusProvider,
   EventStoreBusConfig,
   EventStoreSubscriptionType,
 } from 'nestjs-eventstore';
 
-import { CustomEventHandlers } from './events/handlers';
+//linking of events from EventStore to local events
+const EventInstantiators = [
+  SomeEvent: (_id: any, data: any, loggedInUserId: any) => new SomeEvent(_id, data, loggedInUserId);
+];
 
-const eventStoreBusConfig: EventStoreBusConfig = {
+export const eventStoreBusConfig: EventStoreBusConfig = {
   subscriptions: [
-    {
+    { // persistanct subscription
+      type: EventStoreSubscriptionType.Persistent,
+      stream: '$ce-persons',
+      persistentSubscriptionName: 'contacts',
+    },
+    { // Catchup subscription
       type: EventStoreSubscriptionType.CatchUp,
-      stream: '$ce-identities',
+      stream: '$ce-users',
     },
   ],
   eventInstantiators: {
-    EventName: data => new ImplementedEventName(data),
-    ...
-    ...
+    ...EventInstantiators
   },
-  eventHandlers: CustomEventHandlers,
 };
 
 @Module({
   imports: [
-    EventStoreModule.forRootAsync({
-      useFactory: async (config: ConfigService) => {
-        return {
-          connectionSettings: config.get('eventstore.connectionSettings'),
-          endpoint: config.get('eventstore.tcpEndpoint'),
-        };
+    ConfigModule.load(path.resolve(__dirname, 'config', '**/!(*.d).{ts,js}')),
+    EventStoreCqrsModule.forRootAsync(
+      {
+        useFactory: async (config: ConfigService) => {
+          return {
+            connectionSettings: config.get('eventstore.connectionSettings'),
+            endpoint: config.get('eventstore.tcpEndpoint'),
+          };
+        },
+        inject: [ConfigService],
       },
-      inject: [ConfigService],
-    }),
+      eventStoreBusConfig,
+    ),
   ],
 })
+export class AppModule {}
+
+```
+
+**custom.command.handler.ts**
 
-export class AppModule {
+This following is a way to use the command handlers that push to the custom eventBus to the eventstore using aggregate root.
+
+```typescript
+import { ICommandHandler, CommandHandler } from '@nestjs/cqrs';
+import { SomeCommand } from '../impl/some.command';
+import { EventPublisher } from 'nestjs-eventstore'; //this is necessary as it overrides the default publisher
+import { ObjectAggregate } from '../../models/object.aggregate';
+
+@CommandHandler(SomeCommand)
+export class SomeCommandHandler
+  implements ICommandHandler<SomeCommand> {
+  constructor(private readonly publisher: EventPublisher) {}
+
+  async execute(command: SomeCommand) {
+    const { object, loggedInUserId } = command;
+    const objectAggregate = this.publisher.mergeObjectContext(
+      new ObjectAggregate(object._id, object),
+    );
+    objectAggregate.add(loggedInUserId);
+    objectAggregate.commit();
+  }
 }
 
+```
+
+
+#### Using the EventStoreModule
```

Comment:
We should have this first in the Readme because it's the more casual and unopinionated approach of using this package.
The CQRS Module is more complex and a less common usecase.